src/components/layout/nav/NavigationTabOnLogin.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./NavigationTabOnLogin.scss";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import { NavLink, useHistory } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { logOut, popUpStatus } from "../../redux/actions/index";
+
+const NavigationTabOnLogin = () => {
+  const history = useHistory();
+  const isAdmin = useSelector((state) => state.authentication.isAdmin);
+  const dispatch = useDispatch();
+  const loggingOut = () => {
+    dispatch(logOut());
+    isAdmin && history.push("/");
```

Comment:
do not do this

Reply:
Okay, I'm changing it to normal `if` statement.

Reply:
please don't do this again, if you need further reasoning you may consult the below

https://stackoverflow.com/questions/34704750/one-line-if-vs-in-javascript

Reply:
I see, then I'm gonna use it only for `boolean` values and in `return` statement from now onwards.